### PR TITLE
Sidebar markup and style updates

### PR DESCRIPTION
**Pull request by alexcarpenter**
**Fix search input horizontal spacing**

Screen Shot 2021-06-04 at 8 23 01 AM

image

**Ensure the whole link is clickable**

sidebar-link-hover

**`<li>` should be the only direct child of a `ul`**

image

### CODE REVIEW

File: page.jsx [hunk 1]
```diff
@@ -92,24 +92,24 @@ export default function createPage(swingsetOptions = {}) {
       : componentNames
 
     return (
-      <div className={s.root}>
+      <div className={s.sidebar}>
```
Comment (BRKalow) [inline page.jsx:95]: I _think_ this is wrapping the entire page with the `s.sidebar` class. We might want to retain the `s.root` wrapper here and wrap only the sidebar content with `s.sidebar`.

**Review by BRKalow — COMMENTED** [review]
Top-notch, thanks for the contribution!

File: page.jsx (hunk 1 above)
Comment (alexcarpenter) [reply page.jsx]: @BRKalow you are definitely right. Ran into some issues running the project example locally to see the update but the latest updates seemed to fix it. So I brought back the root class 👍🏼

**Review by jescalan — APPROVED** [review]

**Comment by jescalan** [comment]
Thanks so much for the contribution @alexcarpenter! 👏👏👏👏

**Pull request merged**